packages/before-unload/README.md

```diff
@@ -0,0 +1,27 @@
+# @hooks/before-unload
+
+React hook to provide a message before the page is unloaded
```

Comment:
Just to be a bit clearer, would it be worth adding that it will show in an alert?
```suggestion
React hook to provide a message in an `alert` before the page is unloaded
```

Reply:
I've had a look at the spec for `beforeUnload` and I can't see anything about an alert but it does mention the browser should 'prompt' the user so I'll use your words with a little tweak 👍